docker/docker-compose.1804.53.yaml

```diff
@@ -0,0 +1,37 @@
+version: "3"
+
+services:
+
+  runtime-setup:
+    image: swift-nio-http2:18.04-5.3
+    build:
+      args:
+        ubuntu_version: "bionic"
+        swift_version: "5.3"
```

Comment:
this one can't be right. Did you try the `docker-compose` command? Here I think you need to use `base_image=swiftlang/swift:...` to get the nightlies